projects/core/src/anonymous-consents/config/default-anonymous-consents-config.ts

```diff
@@ -0,0 +1,5 @@
+import { AnonymousConsentsConfig } from './anonymous-consents-config';
+
+export const defaultAnonymousConsentsConfig: AnonymousConsentsConfig = {
+  anonymousConsents: { footerLink: true },
```

Comment:
Should this be configurable?
Like when we enable this feature as a featureFlag in the feature config, does the customer want to remove the link to the consent because how would a user remove their consent if the footer is disabled?

Reply:
Discussed offline